app/views/users/index.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Users list</h1>
\ No newline at end of file
+<h1>The users List :</h1> 
+<div>
+  <% @users.each do |user| %>
+    <div class='user-container'>
+      <%= image_tag user.photo, :class => 'user-photo', alt: 'User photo' %>
+      <h2><%= user.name %></h2>
+      <p>Number of posts: <%= user.PostsCounter.nil? ? 0 : user.user_posts %></p>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Following the project requirements, clicking on each user should take to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png). Try to implement that 😃.